haas/client/switch.py

```diff
@@ -0,0 +1,59 @@
+import json
+from haas.client.base import ClientBase
+from haas.client import errors
+
+
+class Switch(ClientBase):
+    """Consists of calls to query and manipulate node related
+
+    objects and relations.
+    """
+
+    def list(self):
+        """List all nodes that HIL manages """
+        url = self.object_url('/switches')
+        return self.check_response(self.s.get(url))
+
+    def register(self, switch, subtype, *args):
+        """Registers a switch with name <switch> and
+        model <subtype> , and relevant arguments  in <*args>
+        """
+#       It is assumed that the HIL administrator is aware of
```

Comment:
We should do something like `raise NotImplemented` if the function doesn't do what it claims